src/components/CookieMessage/_cookie-message.scss

```diff
@@ -4,12 +4,13 @@
   color: var(--colour-black);
   left: 0;
   letter-spacing: var(--body-letter-spacing);
-  padding: calc(4 * var(--space-unit)) calc(6 * var(--space-unit));
+  padding: var(--space-lg);
```

Comment:
This padding is overridden on line 13, which is fine, but it's worth remembering that `--space-lg` will apply padding across the 3 main device widths

the CSS would be more efficient if line 7 was simply `padding: calc(2 * var(--space-unit));`

Reply:
very good point - I forgot the CSS vars are transpiled down, so thought the method I used wouldn't cause any issues!